### DISCUSSION TITLE
Demo scenes e animações em JS

### DISCUSSION THREAD

**Discussion by dudouz** [frontendbr/forum#2210] [Discussão]
# Demo scenes e animações em JS

Salve pessoal, faz horas que eu quero levantar esse assunto e me perdoem já com antecedência se isso já foi discutido anteriormente.

Uns tempos atrás eu escutei um podcast sobre programação criativa e levantaram o assunto das demoscenes e as práticas que geraram esse movimento underground. Pra quem não associou diretamente, seriam aquelas animações em pixel art que precedem a instalação de games / softwares crackeados.

Após breve pesquisa também notei que tem um nome específico para essas intros: "Cracktro"

Existem desde simples animações de tipografia / letreiros passando pela tela até fractais e coisas bem malucas (que são as que eu tenho interesse em explorar através de algorítimos)....

[https://pt.wikipedia.org/wiki/Crack_intro](https://pt.wikipedia.org/wiki/Crack_intro)

Essa semana que passou eu vi uma publicação no linkedin de um desenvolvedor que mostrava uma animação tipo "synthwave" / demoscene:

[https://victorqribeiro.github.io/retroSynthwave/](https://victorqribeiro.github.io/retroSynthwave/)

E isso me despertou um interesse de tentar reproduzir esse tipo de animação demoscene utilizando o JS.

Alguém por aqui já tentou algo do tipo ou tem algum pensamento a compartilhar?

**Comment by ninetails** [comment]
Lembrei desse vídeo, talvez relacionado:

https://youtu.be/6avJHaC3C2U
